### ISSUE TITLE
Quick start instructions for CMake do not work for gmock.

### ISSUE THREAD

**Issue by cmdremily** [google/googletest#3877]
**Describe the bug**

Following the steps in https://google.github.io/googletest/quickstart-cmake.html will produce a project setup where gmock isn't usable.

**Steps to reproduce the bug**

Minimal reproducible example complete with repro steps in:
https://github.com/cmdremily/gtest-cmake-bug-repro

**Does the bug persist in the most recent commit?**

Yes, as of commitish: 0320f517fd920866d918e564105d68fd4362040a

**What operating system and version are you using?**

I have repro on:
* Windows 10 
* WSL 

**What compiler and version are you using?**

MSVC 2022 Community (CL version: 14.30.30705) 

and 

```
c++ --version
clang version 10.0.0-4ubuntu1
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

**What build system are you using?**

Under Windows 10

```
cmake --version
cmake version 3.23.2

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```

and under WSL

```
cmake --version
cmake version 3.16.3

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```

**Additional context**

Seems to be an issue in the CMakeFile of googletest that only adds `googletest/include` without adding `googlemock/include` to the executable.

**Comment by dreamingskey** [comment]
Ok I'm busy if you guys under stand my creative rights to create and any
coding used in my rights are the rights of mine
Hide quoted tex

**Comment by dinord** [comment]
tldr: The library target for googlemock/include is `GTest::gmock_main`.

GoogleTest and GoogleMock used to be two separate libraries. The directories and CMakeLists files still reflect that structure.  
We have no plans to add a library target that provides both directories, but would consider accepting such a patch.

**Comment by cmdremily** [comment]
The provided comment is not a substitute for correct documentation.

Even if you don't provide a library target, the documentation should be updated to make the _combined_ gmock and gtest framework usable by following the instructions.
